### PR TITLE
Explain main functions in readme

### PR DESCRIPTION
**Pull request by Khay-EMMA**

### CODE REVIEW

**Review by pcatana — CHANGES_REQUESTED** [review]
I think you should not copy the comments from the files and paste them here. The reader can read them directly from the JS file if they want to understand the functions. 
What I think you should do is make an overview of the main functionality of each module and explain it in a paragraph so the reader gets the bigger picture. If the reader wants to know more, he then goes to the actual file and reads the functions and its comments.

**Comment by Khay-EMMA** [comment]
> I think you should not copy the comments from the files and paste them here. The reader can read them directly from the JS file if they want to understand the functions.
> What I think you should do is make an overview of the main functionality of each module and explain it in a paragraph so the reader gets the bigger picture. If the reader wants to know more, he then goes to the actual file and reads the functions and its comments.



> I think you should not copy the comments from the files and paste them here. The reader can read them directly from the JS file if they want to understand the functions.
> What I think you should do is make an overview of the main functionality of each module and explain it in a paragraph so the reader gets the bigger picture. If the reader wants to know more, he then goes to the actual file and reads the functions and its comments.

Okay, got that.

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The Clipper module is responsible for auctioin inititation. It listens for activ
 
 ### transact.js
 
-The Transact module hsndles calculation of gas costs and the signing and sending of transactions. It also has a major class called `GemoetricGasPrice` which is used to get a geometric gas price that increases geometrically.
+The Transact module hsndles calculation of gas costs and the signing and sending of transactions. It also has a major class called `GemoetricGasPrice` which is used to get a geometric gas price that increases geometrically with respect to time and the fixed coefficient
```
Comment (pcatana) [inline README.md:38]: fix typo `hsndles `

**Review by pcatana — CHANGES_REQUESTED** [review]
Please run through the whole readme and fix any grammar issue and typos. I've seen a couple.

**Comment by pcatana** [comment]
LGTM

**Pull request merged**